docs/documentation/release_notes/topics/26_3_0.adoc

```diff
@@ -45,3 +45,8 @@ Read more in https://www.keycloak.org/server/update-compatibility#rolling-update
 = Brokering a OAuth v2 compliant authorization server
 
 In this release, a new broker is available to integrate with any OAuth2 compliant Authorization Server. For more information, see the link:{adminguide_link}#_identity_broker_oauth[OAuth v2 identity providers] chapter in the {adminguide_name}.
+
+= Trusting email verification when brokering OpenID Connect Providers
+
+In this release, the OpenID Connect broker was updated to trust email verification from OpenID Providers based on the `email_verified` claim. If this claim is set to `true`,
```

Comment:
Should this note specify that this is the case just *Trust email* switch is enabled on the IDP? As that is the behaviour of current PR, but this release note does not mention that *Trust email* is pre-requisite, which may confuse users IMO.